src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -1745,8 +1871,12 @@ void AutomationInstrumentClipView::auditionPadAction(int32_t velocity, int32_t y
 					instrumentClipView.sendAuditionNote(false, yDisplay, 64, 0);
 				}
 			}
```

Comment:
added code to re-render display on 7SEG as it seems the auditioned note gets stuck